src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figure = new FigureSupplier();
+        Figure[] array = {figure.getRandomFigure(), figure.getRandomFigure(),
```

Comment:
It's hardcoding, think about how to resolve task description in other way: The first half of figures in this array should be generated with random parameters.